.github/workflows/release.yml

```diff
@@ -62,47 +62,85 @@ jobs:
           path: rel-notes.md
 
   build_release:
-    needs: release_notes
     name: Build Release
+    needs: release_notes
     strategy:
       matrix:
         platform:
           - name: virtual
-            image: default
-            nodes: [self-hosted, 1ES.Pool=gha-virtual-ccf-sub]
+            os: ubuntu
+            image: ghcr.io/microsoft/ccf/ci/default:build-14-01-2025
+            package_version: DEB
+            test_filter: "benchmark|unit"
+          - name: snp
+            os: ubuntu
+            image: ghcr.io/microsoft/ccf/ci/default:build-14-01-2025
+            package_version: DEB
+          - name: virtual
+            os: azure-linux
+            image: mcr.microsoft.com/azurelinux/base/core:3.0
+            package_version: RPM
+            test_filter: "benchmark|unit|protocolstest|lts"
           - name: snp
-            image: default
-            nodes: [self-hosted, 1ES.Pool=gha-virtual-ccf-sub]
-    runs-on: ${{ matrix.platform.nodes }}
+            os: azure-linux
+            image: mcr.microsoft.com/azurelinux/base/core:3.0
+            package_version: RPM
+
+    runs-on: [self-hosted, 1ES.Pool=gha-virtual-ccf-sub]
     container:
-      image: ghcr.io/microsoft/ccf/ci/${{ matrix.platform.image }}:build-14-01-2025
-      options: "--user root --publish-all --cap-add NET_ADMIN --cap-add NET_RAW --cap-add SYS_PTRACE -v /lib/modules:/lib/modules:ro ${{ matrix.platform.container_options }}"
+      image: ${{ matrix.platform.image }}
+      options: "--user root --publish-all --cap-add NET_ADMIN --cap-add NET_RAW --cap-add SYS_PTRACE -v /lib/modules:/lib/modules:ro"
```

Comment:
I believe the modules mount was there for sgxinfo, and is probably not useful anymore.